src/components/Searcher.js

```diff
@@ -440,6 +446,7 @@ class Searcher extends React.Component<SearcherDefaultProps, SearcherProps, Sear
       restParams.set('abc.enabled', ['true']);
       restParams.set('searchProfile', profiles);
     }
+    restParams.set('q.maxresubmits', [`${this.props.maxResubmits}`]);
```

Comment:
Question: is it true that this property should always be set? I.e., is there a use case for not setting it at all?